### PR TITLE
Update windows-autopilot-scenarios.md

### PR DESCRIPTION
**Pull request by okieselbach**
markup links are broken because they are inline of the table tag.... changed them to html now they are functional

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Please advise regarding the Markdown convention used in this PR. Thanks.

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Please confirm whether the Markdown style employed in the commit is compatible with Markdown conventions of rest of the repository. Thanks.

**Comment by JohanFreelancer9** [comment]
@e0i Markdown style is fine and is compatible with conventions of the rest of the repository.

File: windows/deployment/windows-autopilot/windows-autopilot-scenarios.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ The following Windows Autopilot scenarios are described in this guide:
 
 <table>
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]: | Scenario | More information |
| --- | --- |
| Deploy devices that will be set up by a member of the organization and configured for that person | [Windows Autopilot user-driven mode](user-driven.md)|
| Deploy devices that will be automatically configured for shared use, as a kiosk, or as a digital signage device.| [Windows Autopilot self-deploying mode](self-deploying.md)|
| Re-deploy a device in a business-ready state.| [Windows Autopilot Reset](windows-autopilot-reset.md)|
| Pre-provision a device with up-to-date applications, policies and settings.|[White glove](white-glove.md)|
| Deploy Windows 10 on an existing Windows 7 or 8.1 device | [Windows Autopilot for existing devices](existing-devices.md)|

Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]:
```suggestion
```

Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]:
```suggestion
| Scenario | More information |
```

File: windows/deployment/windows-autopilot/windows-autopilot-scenarios.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ The following Windows Autopilot scenarios are described in this guide:
...
 <th>Scenario<th>More information
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]:
```suggestion
| --- | --- |
```

File: windows/deployment/windows-autopilot/windows-autopilot-scenarios.md [hunk 3]
```diff
@@ -28,11 +28,11 @@ The following Windows Autopilot scenarios are described in this guide:
...
-<tr><td>Deploy devices that will be set up by a member of the organization and configured for that person<td>[Windows Autopilot user-driven mode](user-driven.md)
-<tr><td>Deploy devices that will be automatically configured for shared use, as a kiosk, or as a digital signage device.<td>[Windows Autopilot self-deploying mode](self-deploying.md)
-<tr><td>Re-deploy a device in a business-ready state.<td>[Windows Autopilot Reset](windows-autopilot-reset.md)
-<tr><td>Pre-provision a device with up-to-date applications, policies and settings.<td>[White glove](white-glove.md)
-<tr><td>Deploy Windows 10 on an existing Windows 7 or 8.1 device<td>[Windows Autopilot for existing devices](existing-devices.md)
+<tr><td>Deploy devices that will be set up by a member of the organization and configured for that person<td><a href="https://docs.microsoft.com/windows/deployment/windows-autopilot/user-driven.md">Windows Autopilot user-driven mode</a>
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]:
```suggestion
| Deploy devices that will be set up by a member of the organization and configured for that person | [Windows Autopilot user-driven mode](user-driven.md) |
```

File: windows/deployment/windows-autopilot/windows-autopilot-scenarios.md [hunk 4]
```diff
@@ -28,11 +28,11 @@ The following Windows Autopilot scenarios are described in this guide:
...
+<tr><td>Deploy devices that will be automatically configured for shared use, as a kiosk, or as a digital signage device.<td><a href="https://docs.microsoft.com/windows/deployment/windows-autopilot/self-deploying.md">Windows Autopilot self-deploying mode</a>
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]:
```suggestion
| Deploy devices that will be automatically configured for shared use, as a kiosk, or as a digital signage device.| [Windows Autopilot self-deploying mode](self-deploying.md) |
```

File: windows/deployment/windows-autopilot/windows-autopilot-scenarios.md [hunk 5]
```diff
@@ -28,11 +28,11 @@ The following Windows Autopilot scenarios are described in this guide:
...
+<tr><td>Re-deploy a device in a business-ready state.<td><a href="https://docs.microsoft.com/windows/deployment/windows-autopilot/windows-autopilot-reset.md">Windows Autopilot Reset</a>
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]:
```suggestion
| Re-deploy a device in a business-ready state.| [Windows Autopilot Reset](windows-autopilot-reset.md) |
```

File: windows/deployment/windows-autopilot/windows-autopilot-scenarios.md [hunk 6]
```diff
@@ -28,11 +28,11 @@ The following Windows Autopilot scenarios are described in this guide:
...
+<tr><td>Pre-provision a device with up-to-date applications, policies and settings.<td><a href="https://docs.microsoft.com/windows/deployment/windows-autopilot/white-glove.md">White glove</a>
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]:
```suggestion
| Pre-provision a device with up-to-date applications, policies and settings.| [White glove](white-glove.md) |
```

File: windows/deployment/windows-autopilot/windows-autopilot-scenarios.md [hunk 7]
```diff
@@ -28,11 +28,11 @@ The following Windows Autopilot scenarios are described in this guide:
...
+<tr><td>Deploy Windows 10 on an existing Windows 7 or 8.1 device<td><a href="https://docs.microsoft.com/windows/deployment/windows-autopilot/existing-devices.md">Windows Autopilot for existing devices</a>
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]:
```suggestion
| Deploy Windows 10 on an existing Windows 7 or 8.1 device | [Windows Autopilot for existing devices](existing-devices.md) |
```

File: windows/deployment/windows-autopilot/windows-autopilot-scenarios.md [hunk 8]
```diff
@@ -28,11 +28,11 @@ The following Windows Autopilot scenarios are described in this guide:
...
 </table>
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-scenarios.md]:
```suggestion
```

**Comment by e0i** [comment]
@greg-lindsay The PR that has passed copy editing review is ready for technical review. Thanks.

**Comment by e0i** [comment]
@greg-lindsay Following up for updates.

**Comment by e0i** [comment]
@greg-lindsay Following up for updates.

**Pull request merged**